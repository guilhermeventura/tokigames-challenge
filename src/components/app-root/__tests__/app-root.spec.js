/**
 * App Root Test Suite
 *
 * @description Describes the test suite for the AppRoot component
 */

import React from "react";
import { shallow } from "enzyme";
import AppRoot from "../index";

describe("<AppRoot />", () => {
  it("should render without crashing", () => {
    const component = shallow(<AppRoot />);

    expect(component).toMatchSnapshot();
  });
});
