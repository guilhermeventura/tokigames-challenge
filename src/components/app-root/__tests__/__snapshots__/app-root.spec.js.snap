// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AppRoot /> should render without crashing 1`] = `ShallowWrapper {}`;
